templates/partial/_standard-navigation.html

```diff
@@ -0,0 +1,74 @@
+<header id="standard-navigation" class="p-navigation is-dark">
+  <div class="row">
+    <div class="p-navigation__banner col-3 u-no-padding--left">
+      <div class="p-navigation__tagged-logo">
+        <a class="p-navigation__link" href="/">
+          <div class="p-navigation__logo-tag">
+            <img class="p-navigation__logo-icon" src="https://assets.ubuntu.com/v1/82818827-CoF_white.svg" alt="Canonical" width="95">
+          </div>
+          <span class="p-navigation__logo-title">Canonical</span>
+        </a>
+      </div>
+      <a href="#standard-navigation" class="p-navigation__toggle--open p-global-nav-titles" title="menu">Menu</a>
+      <a href="#standard-navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+    </div>
+
+    <nav class="p-navigation__nav col-9" aria-label="main-navigation">
```

Comment:
same here, why using dash in aria-label?

Reply:
Updated